scss/components/table.scss

```diff
@@ -82,20 +85,31 @@ $block: #{$fd-namespace}-table;
             }
         }
     }
-    tbody {
```

Comment:
I think `__fixed-col` a couple lines above should also be changed to `__fixed-cell`, but I could be getting confused on everything moving around.

Reply:
__fixed_col refers to an entire column that is fixed, so in this case it seems to be described correctly?

Reply:
Yeah I'm fine leaving this if it was already thought through as a "column" style.

Reply:
I ended up switching it to cell - it seems like its styling the individual cells rather than a whole, so thought better to change to keep consistent with our previous change to fd-table__cell